tests/tools/selectv_rcode.c

```diff
@@ -70,27 +93,44 @@ static char *type_to_str(int type)
             return "updater";
         case SELECTV_UPDATER_THREAD:
             return "selectv_updater";
+        case NOSELECT_UPDATER:
+            return "noselect_updater";
+        case SINGLE_STATEMENT_NOSELECT_UPDATER:
+            return "single_statement_noselect_updater";
+        case POINT_IN_TIME_UPDATER:
+            return "point_in_time_updater";
+        case INTRANS_EFFECT_UPDATER:
+            return "intrans_effect_updater";
         default:
             abort();
             break;
     }
 }
 
+int is_common_acceptable_error(cdb2_hndl_tp *db, int rc)
+{
+    switch(rc) {
+        /* Master lost transaction */
+        case -109:
+        case 203:
```

Comment:
CDB2ERR_DEADLOCK instead of 203 .. -109 is not referenced in cdb2api.h (maybe a different pr)